### DISCUSSION TITLE
Lack of conformity at tutorials

### DISCUSSION THREAD

**Discussion by Erickrk** [microsoft/AirSim#4334] [Ideas]
Hello everyone!

Very nice to be finally interacting with the community after a while using AirSim.
My question is related to a guideline shown at tutorials. I was simply trying to build the Blocks environment on a new computer and went for the tutorials to do so.

There I found two mismatched pieces of information:
a) [Build on Windows](https://microsoft.github.io/AirSim/build_windows/) page says "Select your Unreal project as Start Up project (for example, Blocks project) and make sure Build config is set to "Develop Editor" and x64."
b) While the [Setup Blocks Environment ](https://microsoft.github.io/AirSim/unreal_blocks/) page says "Make sure Blocks project is the startup project, build configuration is set to DebugGame_Editor and Win64. Hit F5 to run."

Is there any significant difference among those? I'm reporting because someone looking at these two for the first time might get a bit confused on what to do.

**Comment by jonyMarino** [comment]
Hi @Erickrk! Thanks for the report. If you want to debug your blocks project, you should select the Debug option; otherwise, the Develop one

**Reply by Erickrk** [reply]
It makes sense, thanks for the reply!
